misoc/integration/cpu_interface.py

```diff
@@ -12,6 +12,10 @@ def get_cpu_mak(cpu):
         triple = "or1k-linux"
         cpuflags = "-mhard-mul -mhard-div -mror -mffl1 -maddc"
         clang = "1"
+    if cpu == "vexriscv":
+        triple = "riscv64-unknown-elf" # GCC from https://static.dev.sifive.com/dev-tools/riscv64-unknown-elf-gcc-20171231-x86_64-linux-centos6.tar.gz
+        cpuflags = "-D__vexriscv__ -march=rv32im  -mabi=ilp32 -nostdlib -lgcc -mcmodel=medany -nostartfiles -ffreestanding -Wl,-Bstatic"
```

Comment:
All these flags for freestanding compilation should go to the Makefile (and most of them are already there). `-lgcc` shouldn't appear at all, I think. `-Wl,-Bstatic` is likely unnecessary.